isobar/pattern/fade.py

```diff
@@ -107,10 +115,15 @@ class PFadeNotewiseRandom(PFadeNotewise):
     """ PFadeNotewiseRandom: Fade a pattern in/out by gradually introducing random notes. """
 
     def __init__(self, *args, **kwargs):
+        self.args = args
```

Comment:
Just spotted these new additions - what are they for?

Reply:
Saved the args so they could be used in the `repr` later on to easily reproduce the object. I'm realizing this just is a subclass of `PFadeNotewise` so would we wanna remove the `repr` function altogether for this class and have it default to the regular `PFadeNotewise` `repr`?

Reply:
ah I see! I think actually my implementation of `PFadeNotewiseRandom` was just a bit lazy, and should be corrected to be the same as `PFadeNotewise` :-)

Both of these PFade* classes are somewhat legacy and could do with revisiting someday...

Reply:
I toyed around with `PFadeNotewiseRandom` to give it the proper params and map those properly to the `repr` function, let me know if you'd want me to adjust this back of if this is a decent solution for now